var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -510,6 +514,8 @@ def enable_or_disable(variant, keyword="USE", var=None):
         # Build in parallel to speed up build times
         env.set("MAX_JOBS", make_jobs)
 
+        env.set("USE_CCACHE", "OFF")
```

Comment:
What is this and why is it needed?

Reply:
It avoids `ccache` pollution due to `ccache <compiler wrapper>`. The compiler wrapper should be responsible for calling `ccache <effective compiler invocation>`.

Reply:
Generic fix: https://github.com/spack/spack/pull/45275